requirements.txt

```diff
@@ -2,4 +2,5 @@ numpy >=1.15
 pandas
 boto3
 s3fs
-pylint
\ No newline at end of file
+pylint
+jupyter
```

Comment:
This is more of my fault tbh, but in https://github.com/tagr-dev/tagr/pull/5, I realized that i didn't include black in our requirements.txt. Could you add that for me pls